Sources/PackageManagerDocs/Documentation.docc/AddingDependencies.md

```diff
@@ -43,15 +43,47 @@ For more information on resolving package versions, see <doc:ResolvingPackageVer
 
 ### Constraining dependency versions
 
-Constrain the version of a remote dependency when you when you declare the dependency.
-The package manager uses git tags interpretted as semantic versions to identify eligible versions of packages.
+Constrain the version of a remote dependency when you declare the dependency.
+The package manager uses git tags, interpreted as a semantic versions, to identify eligible versions of packages.
 
-> Note: tags for package versions should include all three components of a semantic version: major, minor, and patch. 
+> Note: tags for package versions should include all three components of a semantic version: major, minor, and patch.
 > Tags that only include one or two of those components are not interpreted as semantic versions.
 
 Use the version requirement when you declare the dependency to limit what the package manager can choose.
 The version requirement can be a range of possible semantic versions, a specific semantic version, a branch name, or a commit hash.
-The API reference documentation for [Package.Dependency](https://developer.apple.com/documentation/packagedescription/package/dependency) defines the methods to use.   
+The API reference documentation for [Package.Dependency](https://docs.swift.org/swiftpm/documentation/packagedescription/package/dependency) defines the methods to use.
+
+### Packages with Traits
+
+Traits, introduced with Swift 6.1, allow packages to offer additional API that may include optional dependencies.
+Packages should offer traits to provide API beyond the core of a package.
+For example, a package may provide an experimental API, optional API that requires additional dependencies, or functionality that a developer may want to disable in specific circumstances.
```

Comment:
I think the word `disable` is too strong here. There is no way to disable a trait really since across a package graph multiple different packages might enable different traits and we always take the union of all traits.

Reply:
Super useful detail, thank you! I'm adding a short paragraph explaining that detail a bit more for clarity, as it wasn't obvious to me from earlier reading.